package.json

```diff
@@ -145,7 +146,7 @@
 		"file-loader": "^6.0.0",
 		"friendly-errors-webpack-plugin": "^1.7.0",
 		"git-revision-webpack-plugin": "^3.0.3",
-		"hard-source-webpack-plugin": "^0.13.1",
+		"hard-source-webpack-plugin-fixed-hashbug": "^0.13.2",
```

Comment:
lol

Reply:
Its a fork, someone in the community got tired of the PR sitting around with the fix for 2 years....